editor/CMakeLists.txt

```diff
@@ -13,3 +13,9 @@
 # limitations under the License.
 
 add_subdirectory(app)
+add_subdirectory(analytics)
+add_subdirectory(auth)
+# TODO(cl/383898177): Waiting for this cl to be synced to Github repo.
+# add_subdirectory(crashlytics)
```

Comment:
Do you comment this out purposely?

Reply:
Yes.  Still waiting for Crashlytics team review, and copybara.

That can take awhile so just push the changes for other libraries.